### ISSUE TITLE
Custom compress not giving the expected  image size.

### ISSUE THREAD

**Issue by azizconsoliads** [zetbaitsu/Compressor#202]
I have used it according to the documentation but when am compressing image to a specific size like 2mb it will not compress image exact to 2mb. It will give result like 1.1mb or something like that.

**Comment by rubyna11** [comment]
Did you find the solution?

**Comment by logickoder** [comment]
@azizconsoliads I don't think it is possible to get an exact image size during compression, we can just compress against something but we can't really specify the exact size to get
